app/config/locale/translations/es.json

```diff
@@ -2,31 +2,31 @@
     "settings.inspire": "\"El arte de ser sabio es el arte de saber qué pasar por alto\"",
     "settings.locale": "es",
     "settings.direction": "ltr",
-    "emails.sender": "Equipo %s",
+    "emails.sender": "El equipo de %s",
     "emails.verification.subject": "Verificación de cuenta",
-    "emails.verification.hello": "Hola {{name}}",
-    "emails.verification.body": "Haz clic en este enlace para verificar tu correo.",
+    "emails.verification.hello": "Hola, {{name}}",
+    "emails.verification.body": "Haz clic en este enlace para verificar tu correo:",
     "emails.verification.footer": "Si no has solicitado verificar este correo, puedes ignorar este mensaje.",
-    "emails.verification.thanks": "Gracias",
-    "emails.verification.signature": "Equipo de {{project}}",
+    "emails.verification.thanks": "Gracias.",
+    "emails.verification.signature": "El equipo de {{project}}.",
     "emails.magicSession.subject": "Inicio de sesión",
-    "emails.magicSession.hello": "Hola,",
-    "emails.magicSession.body": "Haz clic en este enlace para iniciar sesión.",
-    "emails.magicSession.footer": "Si no has solicitado ingresar usando este correo, puedes ignorar este mensaje.",
-    "emails.magicSession.thanks": "Gracias",
-    "emails.magicSession.signature": "Equipo de {{project}}",
+    "emails.magicSession.hello": "Hola, {{name}}",
```

Comment:
Looking at EN template, we dont use name here. Maybe it may be missing at this stage. Let's remove it.

Reply:
Done